script.js

```diff
@@ -1,4 +1,155 @@
+// Global variables
+var GAME_STATE_DICE_ROLL = "GAME_STATE_DICE_ROLL";
+var GAME_STATE_CHOOSE_DICE_ORDER = "GAME_STATE_CHOOSE_DICE_ORDER";
+var GAME_STATE_COMPARE_SCORES = "GAME_STATE_COMPARE_SCORES";
+var gameState = GAME_STATE_DICE_ROLL;
+
+var INSTRUCTIONS_FOR_NEXT_GAME_STATE = `Please choose the order of the dice by inputting 1 or 2`;
+
+var currentPlayerRolls = [];
+
+var player1ScoreCounter = 0;
+var player2ScoreCounter = 0;
+
+var currentPlayer = 1;
+var allPlayersScore = [];
+
+var mode = "";
+var regularMode = "regular";
+var lowestMode = "lowest";
+var MODE_SELECTION_INSTRUCTIONS = `please choose a mode to play:<br><br>${regularMode} or ${lowestMode}`;
+
+// Helper Function
+var diceRoll = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+var rollDiceForPlayer = function () {
+  var counter = 0;
+  for (var counter = 0; counter < 2; counter += 1) {
+    currentPlayerRolls.push(diceRoll());
+    console.log("error check " + currentPlayerRolls);
+  }
```

Comment:
great that you've opted for a for loop here.

Also generally, it's best to remove console logs from final codebase, as console log is for you to check stuff internally.